app/src/containers/MenuMobile.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import MenuMobile from '../components/Shared/MenuMobile';
+import { setVisibleMenu } from '../actions/appearence';
+
+const mapStateToProps = (state) => ({
+  menuVisible: state.appearance.menuVisible
```

Comment:
@tiagojsag 
But if that is a component state...who can we change that if we call the function on header?
I mean that is neccesary not?